css/default.scss

```diff
@@ -88,7 +88,6 @@ body {
 
 	background-color: $navbar-bg-color;
 	border-bottom: 1px solid #ccc;
-	box-shadow: 0 0 20px rgba(204,204,204,0.5);
```

Comment:
Any specific reason for this removal?

Reply:
Shoot, forgot to mention this one: this is to prevent the box-shadow of the navbar from leaking through, when scrolling horizontally (you can see this on the gifs i made earlier). It is only visible on desktop when the main area is smaller and the background image comes in, but even there it's almost negligible, in my opinion.

It's this (with box-shadow):

![image](https://user-images.githubusercontent.com/11561719/145463783-6681c4df-20f6-4114-a0b0-18202553d8a2.png)

versus this (no box-shadow):

![image](https://user-images.githubusercontent.com/11561719/145463866-3f0dc822-6823-4c54-8c35-0808d198ed82.png)